bitnami/kong/templates/_helpers.tpl

```diff
@@ -288,7 +288,7 @@ INGRESS AND CASANDRA: Cassandra-backed deployments of Kong managed by Kong Ingre
 Function to validate incompatibilities with deploying Kong as a daemonset
 */}}
 {{- define "kong.validateValues.daemonset" -}}
-{{- if and .Values.useDaemonset (or .Values.pdb.enabled .Values.autoscaling.enabled) -}}
+{{- if and .Values.useDaemonset (or (coalesce .Values.pdb.create .Values.pdb.enabled) .Values.autoscaling.enabled) -}}
```

Comment:
`pdb.enabled` was deprecated in the previous version, I think you can remove it. (or at least set the params in the opposite order)

Reply:
I got confused because `pdb.enabled` was referenced in another template.

I applied your suggestion and removed it from both